app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -9,23 +9,12 @@
 .gem-c-cards__list {
   list-style: none;
   padding: 0;
-  // Remove the outermost left and right margin from the internal margin of the list items.
-  margin: 0 govuk-spacing(-3);
-
+  margin: 0;
   display: grid;
-  grid-auto-flow: row;   // Use CSS grid to calculate the number of rows
-  grid-auto-rows: fractions(1); // Set all rows to same fractional height of the complete grid
-  grid-template-columns: fractions(1);
-
-  @include govuk-media-query($from: "tablet") {
-    grid-template-columns: fractions(2);
-  }
+  grid-column-gap: govuk-spacing(6); // Use deprecated grid-column-gap for Grade C browser support
+  column-gap: govuk-spacing(6);
```

Comment:
Non-blocking: it's probably more efficient in terms of lines of CSS the way it is, but it occurs to me that the grid attributes aren't needed by default, and seem to belong more logically only in the two and three column class variants.

Reply:
Good point, it would definitely be possible to remove the `display: grid` from the `.gem-c-cards__list` and either set this on the `two / three` column classes, or even a separate modifier/utility class specifically for using CSS grid layouts.

Would need to explore this further, can think of lots of pros and cons to each approach.